src/ontology/mondo-edit.obo

```diff
@@ -556240,6 +556240,7 @@ id: MONDO:0957248
 name: developmental and epileptic encephalopathy, 31B
 def: "Any developmental and epileptic encephalopathy in which the cause of the disease is a homozygous mutation in the DNM1 gene." [https://orcid.org/0000-0002-0587-4693, OMIM:620352]
 subset: gard_rare {source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
```

Comment:
@yshwetar : while not "wrong", we do not manually add any rare subset annotations. 
All subsets are maintained automatically. Therefore anything you add manually will be removed when we update the subsets. You can see more information [here](https://mondo.readthedocs.io/en/latest/editors-guide/rare-disease-curator/)

It is ok to leave here, but just keep this in mind for the future.